BlazarTestBase/src/main/java/org/kohsuke/github/BlazarGHRepository.java

```diff
@@ -23,7 +23,7 @@
   private final String name;
   private final String fullName;
   private Map<String, ? extends GHBranch> branches;
-  private final BlazarGHUser owner;
+  private BlazarGHUser owner;
   private List<BlazarGHCommit> commits;
```

Comment:
Because during tests I'm sometimes pushing commits and then reverting them. Those actions modify these fields.